File: javascript/armstrong-numbers/armstrong-numbers.js
export const isArmstrongNumber = (numberInput) => {
	let numberInputParsed = `${numberInput}`;
	let numberOfDigits = numberInputParsed.length;
	let total = 0;

	[...numberInputParsed].forEach(function (digit) { total += digit ** numberOfDigits; });

	return total === numberInput;
};
	

File: csharp/armstrong-numbers/ArmstrongNumbers.cs
using System;
using System.Linq;

public static class ArmstrongNumbers
{
    public static bool IsArmstrongNumber(int number)
    {
        var numberLength = (double)$"{number}".Length;
        var result = $"{number}".Aggregate(0, (total, digit) => total + (int)Math.Pow(char.GetNumericValue(digit), numberLength));

        return result == (double)number;
    }
}